tool/tsh/proxy_test.go

```diff
@@ -587,5 +587,5 @@ func mustFindFailedNodeLoginAttempt(t *testing.T, av []apievents.AuditEvent, nod
 			return
 		}
 	}
-	t.Error("failed to find AuthAttemptFailureCode event")
+	t.Errorf("failed to find AuthAttemptFailureCode event (0/%d events matched)", len(av))
```

Comment:
Unrelated, but this one is also flaky and I thought a little more detail in the failure message wouldn't hurt.

Reply:
I think this one just needs to wait for the event to show up